### PR TITLE
{chem}[foss/2021b] xtb v6.5.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)
See also
- [ ] https://github.com/easybuilders/easybuild-easyconfigs/pull/15187
Some of the suggestions from the maintainers were taken from there. The question is if we really need two versions with the same toolchain?

### CODE REVIEW

File: easybuild/easyconfigs/x/xtb/xtb-6.5.0-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+github_account = 'grimme-lab'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['0b39f6e87ce07abcb0884cda5d3370601b6332ad72c8213e87ffa74ae28895fb']
+
+builddependencies = [
+    ('Meson', '0.58.2'),
+    ('Ninja', '1.10.2'),
+    ('pkg-config', '0.29.2'),
```
Comment (smoors) [inline easybuild/easyconfigs/x/xtb/xtb-6.5.0-foss-2021b.eb:24]: why not use pkgconf here? 
```suggestion
    ('pkgconf', '1.8.0'),
```

**Comment by sassy-crick** [comment]
Would it make sense to promote that to the `foss-2022a` toolchain? When I wrote that PR I was not aware it is already in the development branch. That would also avoid to have two versions in the same toolchain. From what I can see no other software depends on it.

**Comment by emdrago** [comment]
Test report by @emdrago
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
gput052 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz, 1 x NVIDIA NVIDIA GeForce RTX 2080 Ti, 510.47.03, Python 3.9.13
See https://gist.github.com/c67503262f686e9c6df83b8d29f6dd97 for a full test report.

**Comment by smoors** [comment]
> Would it make sense to promote that to the `foss-2022a` toolchain? When I wrote that PR I was not aware it is already in the development branch. That would also avoid to have two versions in the same toolchain. From what I can see no other software depends on it.

it doesn't hurt to have multiple versions in the same toolchain, as long as only 1 is used as dep.

of course, it's always good to have it for the latest toolchain as well, maybe also update the version to v6.5.1 then?
probably better to do it in a new PR though, this one has been sitting here for too long already

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

**Pull request merged**